### PR TITLE
Add command /bot info

### PR DESCRIPTION
**Pull request by sweet235**
Shows the same information as current `listbots` tries to show, but lets a user select a set of bots. This is to give us the the freedom to fix https://github.com/Unvanquished/Unvanquished/issues/2341 in any way we see fit, even by reverting to the old `listbots`.

Edit:
Usage example:
```
]/listbots
listbots: 6 bots in game: 
Slot Name Team [s=skill ss=skillset b=behavior g=goal] 
1 [bot] Morgan: alien [s=5 ss="fighting aim-head aim-barbs movement tyrant-attack-run mara-attack-jump mantis-attack-jump goon-attack-jump feels-pain" b=default g=(723 138 32)] 
2 [bot] Alaric: human [s=5 ss="fighting predict-aim feels-pain medkit flee-run" b=default g=(-865 1803 -6)] 
3 [bot] Soizic: alien [s=5 ss="fighting movement mara-attack-jump goon-attack-jump aim-barbs tyrant-attack-run mantis-attack-jump aim-head feels-pain" b=default g=(909 1984 -8)] 
4 [bot] Baudry: human [s=5 ss="feels-pain buy-armor prefer-armor medkit" b=default g=(-1063 1752 -11)] 
5 [bot] Itsaso: alien [s=5 ss="fighting aim-head movement aim-barbs goon-attack-jump feels-pain mantis-attack-jump tyrant-flee-run mantis-flee-jump" b=default g=(1710 860 127)] 
6 [bot] Ferdinand: human [s=5 ss="feels-pain buy-armor prefer-armor flee-run" b=default g=<invalid>] 

]/bot info 1 3 5
Slot Name Team [s=skill ss=skillset b=behavior g=goal] 
1 [bot] Morgan: alien [s=5 ss="fighting aim-head aim-barbs movement tyrant-attack-run mara-attack-jump mantis-attack-jump goon-attack-jump feels-pain" b=default g=(1432 354 106)]
3 [bot] Soizic: alien [s=5 ss="fighting movement mara-attack-jump goon-attack-jump aim-barbs tyrant-attack-run mantis-attack-jump aim-head feels-pain" b=default g=(player|#4)]
5 [bot] Itsaso: alien [s=5 ss="fighting aim-head movement aim-barbs goon-attack-jump feels-pain mantis-attack-jump tyrant-flee-run mantis-flee-jump" b=default g=(342 2317 -114)]
 ```

Since a user can control the output size, it is less of a problem if the output becomes too big.

### CODE REVIEW

File: src/sgame/sg_admin.cpp [hunk 1]
```diff
@@ -5887,6 +5888,43 @@ static bool BotFillCmd( gentity_t *ent, const Cmd::Args& args )
...
+		char err[ MAX_STRING_CHARS ];
+		botIds[ i - 2 ] = G_ClientNumberFromString( args[ i ].data(), err, sizeof( err ) );
+		if ( botIds[ i - 2 ] < 0 )
+		{
+			ADMP( va( "%s %s %s %s", QQ( "^3$1$:^* $2t$: $3$" ), "bot", Quote( err ), Quote( args[ i ].data() ) ) );
+			return false;
+		}
+	}
+	ADMBP_begin();
+	ADMP( QQ( N_( "Slot Name Team [s=skill ss=skillset b=behavior g=goal]" ) ) );
```
Comment (slipher) [inline src/sgame/sg_admin.cpp:5911]: Should this be ADMBP?

File: src/sgame/sg_admin.cpp [hunk 2]
```diff
@@ -5774,11 +5774,12 @@ void G_admin_cleanup()
 static void BotUsage( gentity_t *ent )
 {
 	static const char bot_usage[] = QQ( N_( "^3bot:^* usage: bot add (<name> | *) (aliens | humans) [<skill level> [<behavior>]]\n"
-	                                        "            bot fill <count> [<team> [<skill level>]]\n"
+	                                        "            bot behavior (<name> | <slot#>) <behavior>\n"
```
Comment (slipher) [inline src/sgame/sg_admin.cpp:5777]: The idea is to put the most used commands at the beginning rather than alphabetize

**Review by slipher — COMMENTED** [review]
So we have the new command but what about reverting listbots?

**Comment by slipher** [comment]
`listbots` should work now. Is there any reason for this now?

**Comment by sweet235** [comment]
> `listbots` should work now. Is there any reason for this now?

That depends on what we want to do. The `listbots` output is quite long, and might grow in the future. It became harder for admins to check current bot behaviors.

I suggest we either use this, or add switches to `listbots` to control the output size.

**Comment by bmorel** [comment]
@slipher 

> listbots should work now. Is there any reason for this now?

In practice, listbots does not work. It technically works, but is unusable: humans can't really parse this efficiently, especially in the heat of the fight.

File: src/sgame/sg_admin.cpp (hunk 1 above)
Comment (bmorel) [reply src/sgame/sg_admin.cpp:5911]: I think this is the only strictly required change. Not changing the order in which description is shown would be nice, but it's much less important than having listbots readable.

**Pull request closed without merging**